config/initializers/_config.rb

```diff
@@ -2,7 +2,10 @@ module RsvpRails
   mattr_accessor :config
 end
 
+fail("Missing CONSTELLATION_URL ENV value") unless ENV['CONSTELLATION_URL']
```

Comment:
The app doesn't make much sense without these, so fail to boot if not set.

Reply:
👌

Reply:
Maybe you can use `ENV#fetch`?

```ruby
ENV.fetch("I_DO_NOT_EXIST")
# => KeyError: key not found: "I_DO_NOT_EXIST"
```

Reply:
Nice, didn’t knew it would raise!